### ISSUE TITLE
Show top bar items in mobile view

### ISSUE THREAD

**Issue by burkfers** [Bastardkb/bastardkb.github.io#3]
The mobile-friendly stylesheet hides elements from the top bar visible in desktop view.

Once #2 is merged, this also hides the dark mode toggle from mobile view.

We should estimate the effort required to show at least that toggle to mobile users and decide whether it's realistic for us to attempt to change.

**Comment by bstiq** [comment]
#2 is merged.

Let's keep the issue here open, but it's low priority.
